common/src/main/scala/org/apache/comet/CometConf.scala

```diff
@@ -286,9 +286,10 @@ object CometConf extends ShimCometConf {
     conf("spark.comet.explainFallback.enabled")
       .doc(
         "When this setting is enabled, Comet will provide logging explaining the reason(s) " +
-          "why a query stage cannot be executed natively.")
+          "why a query stage cannot be executed natively. Set this to false to " +
```

Comment:
Hmmm, not sure the exact difference from `COMET_EXPLAIN_VERBOSE_ENABLED`... @parthchandra

Reply:
That's a good point. I had tested with this set to true, but for the fallback logging we always want the verbose tree format. I will push a commit to resolve this

Reply:
Other than the code that I introduced in this PR, which I am now going to remove, the only calls to `org.apache.comet.ExtendedExplainInfo#generateExtendedInfo` are from tests, so the `COMET_EXPLAIN_VERBOSE_ENABLED` does not actually have any impact on end users. Perhaps it should be marked as an internal config?

Reply:
I just remembered that this `ExtendedExplainInfo` is implementing a Spark API that the Spark UI can call in Spark 4. Setting COMET_EXPLAIN_VERBOSE_ENABLED will show the tree view there.

Reply:
> I just remembered that this `ExtendedExplainInfo` is implementing a Spark API that the Spark UI can call in Spark 4. Setting COMET_EXPLAIN_VERBOSE_ENABLED will show the tree view there.

That is correct. We should not change the default (yet). By default the user will see the abbreviated version in the Spark UI. Setting `COMET_EXPLAIN_VERBOSE_ENABLED` to true will show the verbose version in the Spark UI. 
For our internal logging purposes we can explicitly call `ExtendedExplainInfo.generateVerboseExtendedInfo`